app/templates/application.hbs

```diff
@@ -26,6 +26,8 @@
   {{django-page page=model.splashPage.pagecontent}}
 {{/if}}
 
+{{onboarding-message }}
+
```

Comment:
I think renaming this component to something more descriptive, since this onboarding is really for the new site? Call it something like `jukeboxes-onboarding-message` ?